docs/Commands.html

```diff
@@ -158,8 +156,8 @@ <h3>Command <code>listConfigurations</code></h3>
 
 <h3 id=saveBinaryData>Command <code>saveBinaryData</code></h3>
 <p>
-This is used to save Shasta binary data.
-Binary data are stored in directory <code>Data</code> in the assembly directory.
+This command is used to save Shasta binary data.
+Shasta stores binary data in directory <code>Data</code> in the assembly directory.
 This command makes a copy on disk in <code>Data</code>. 
```

Comment:
Actually the copy on disk is named `DataOnDisk`, not `Data`. This error was already there before these changes.

Reply:
Ok. I will update it with DataOnDIsk